src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
+// Working with data
+// Making array from textContant
 
-// write your code here
+const populationData = Array.from(document.querySelectorAll('span.population'));
```

Comment:
Ensure that the `document.querySelectorAll('span.population')` selects the correct elements. If the elements are not found, `populationData` will be an empty array, which might lead to unexpected results in subsequent operations.